pkg/util/pki/csr.go

```diff
@@ -57,6 +58,22 @@ func DNSNamesForCertificate(crt *v1alpha1.Certificate) []string {
 	return crt.Spec.DNSNames
 }
 
+func IPAddressesForCertificate(crt *v1alpha1.Certificate) []net.IP {
+	var ipAddresses []net.IP
+	for _, ip := range IPAddressesNameForCertificate(crt) {
+		ipAddresses = append(ipAddresses, net.ParseIP(ip))
```

Comment:
As per the docs, ParseIP will return nil if the input data is invalid: https://golang.org/pkg/net/#ParseIP

Whilst you have implemented validation checks for this field when the resource is admitted, we should still err on the side of caution here and handle the case where an invalid IP address is passed in 😄